test/hamiltonian_consistency.jl

```diff
@@ -32,21 +28,21 @@ include("testcases.jl")
         ρ_trial = compute_density(basis, ψ_trial, occupation)
 
         @assert length(basis.terms) == 1
-        E0, ops = ene_ops(basis.terms[1], ψ, occupation; ρ=ρ)
-        E1, _ = ene_ops(basis.terms[1], ψ_trial, occupation; ρ=ρ_trial)
-        diff = (E1 - E0)/ε
+        E0, ham = energy_hamiltonian(basis, ψ, occupation; ρ=ρ)
+        E1, _ = energy_hamiltonian(basis, ψ_trial, occupation; ρ=ρ_trial)
+        diff = (E1.total - E0.total)/ε
```

Comment:
Why do you need this? Because for the anyons an energy term may return two hamiltonian terms?

Reply:
Yes, and it's cleaner

Reply:
ok.